build: update bazel to v1.1.0

devversion opened the pull request:
Updates Bazel to the latest stable version. Bazel 1.1.0
supposedly fixes a permission bug in Windows. Hence we
should try to update and see if that fixes the bug.

It's generally good to be up to date. See potential bug
fix commit: https://github.com/bazelbuild/bazel/commit/618e5a28f7f735c37724377b15775a4975349c74.

devversion committed:
build: update bazel to v1.1.0

Updates Bazel to the latest stable version. Bazel 1.1.0
supposedly fixes a permission bug in Windows. Hence we
should try to update and see if that fixes the bug.

It's generally good to be up to date. See potential bug
fix commit:
https://github.com/bazelbuild/bazel/commit/618e5a28f7f735c37724377b15775a4975349c74.

kyliau commented:
@devversion Could you also please update https://github.com/angular/angular/blob/b30bb8dd91c61684f16664e6ae6589113737a76d/packages/bazel/src/schematics/ng-add/index.ts#L51 ?

devversion committed:
feat(bazel): update ng-add to use bazel v1.1.0

The ng-add schematic for `@angular/bazel` should set up
the latest Bazel version v1.1.0 that contains fixes for
windows. e.g. https://github.com/bazelbuild/bazel/commit/618e5a28f7f735c37724377b15775a4975349c74

devversion commented:
@kyliau Done. Pushed a separate commit since this separate from `build`-only changes. Please have another look.

kyliau reviewed (approved):
thank you!

josephperrott reviewed (approved):
LGTM
